### PR TITLE
Imported Grade-book view in CCX Dashboard tab.

### LINKED ISSUE #58: POC Gradebook doesn't display POC Dashboard tabs
As POC Coach, when I select the POC Gradebook tab, the POC Dashboard tabs disappear. The URL is:
https://sandbox2o.mitx.mit.edu/courses/MITx/4.605x_2/3T2014/poc_gradebook 

I'd expect some indication that I was still in the POC Dashboard, and which POC I was viewing the Gradebook for.

poc_gradebook

### PR DESCRIPTION
**Pull request by amir-qayyum-khan**
Hi

In this PR I Imported CCX grade book in coach dashboard and section **Student admin** .
This is to avoid multiple clicks.

@pdpinch @carsongee @pwilkins 
closes #58

screen shot 2015-09-30 at 4 11 19 pm
screen shot 2015-09-30 at 4 11 29 pm

### CODE REVIEW

**Comment by pdpinch** [comment]
This is a lot nicer, but we probably shouldn't do this until we have pagination. This page has the potential to take a very long time to load with long course that have many students.

**Comment by pdpinch** [comment]
Also, I'm concerned about the CSS. It seems very generic for the limited scope.

**Comment by amir-qayyum-khan** [comment]
@pdpinch should I go for another approach or should wait for pagination PR to merge?

**Comment by pdpinch** [comment]
We could push ahead with this without pagination, but I still have concerns about the CSS approach.

**Comment by amir-qayyum-khan** [comment]
@pdpinch  About css we can discuss in detail.

- I am just using existing gradebook with same params/flags to hide its header, footer navigation tabs. Gradebook supports this kind of customisation. Then I am loading this gradebook.html file in a ```div``` in student admin page using ajax.

- To adjust gradebook in new ```div``` inside student admin page I need to override some css properties. So thats why i made addition in css file.

File: lms/static/sass/course/instructor/_instructor_2.scss [hunk 1]
```diff
@@ -159,6 +159,44 @@
     }
   }
 
+  /* Start: Override style for CCX grade book */
+  .container {
```
Comment (pdpinch) [inline lms/static/sass/course/instructor/_instructor_2.scss]: Is this class only used by the CCX display of the gradebook? If so, can we make that clearer, maybe by naming it `.ccx-container` ?

Comment (amir-qayyum-khan) [inline lms/static/sass/course/instructor/_instructor_2.scss]: No this is used by gradebook for both instructor and ccx. Here I am just overriding some properties to support grade book UI in a div inside ccx dashboard

https://github.com/mitocw/edx-platform/pull/90/files#diff-9fb9f7eb54cc9531c7ed284e577e62a5R42

Comment (amir-qayyum-khan) [inline lms/static/sass/course/instructor/_instructor_2.scss]: @pdpinch this is class used by grade book. I am here fixing some css properties to make grade book work under div in side ccx dashboard. This class will be use by both instructor and coach. 

https://github.com/mitocw/edx-platform/pull/90/files#diff-9fb9f7eb54cc9531c7ed284e577e62a5R41

**Comment by pdpinch** [comment]
Sorry this has been sitting so long @amir-qayyum-khan. You can go ahead and open a PR at edX. 

You'll need to clean up the cover letter.

**Comment by amir-qayyum-khan** [comment]
@pdpinch this depends on https://github.com/edx/edx-platform/pull/10353.
I need to make few changes to adapt pagination.

**Comment by amir-qayyum-khan** [comment]
Created upstream PR https://github.com/edx/edx-platform/pull/10637

**Pull request closed without merging**